### ISSUE TITLE
Image Not Found Error - Despite email saying it is on the map

### ISSUE THREAD

**Issue by talllguy** [mapillary/mapillary_issues#1719]
Just up'd a sequence (see https://github.com/mapillary/mapillary_issues/issues/1677#issuecomment-187219998) and was looking to check out the photos on the map. I got this error:

![2016-02-22 10_00_28-mapillary _ crowdsourced street photos](https://cloud.githubusercontent.com/assets/1827881/13221886/89163b32-d94b-11e5-9166-e7453e340831.png)

Oddly, I got an email saying the sequence was on the map, but they don't seem to be viewable yet...
![2016-02-22 10_00_41-inbox elliott plack gmail com](https://cloud.githubusercontent.com/assets/1827881/13221909/9d25b4cc-d94b-11e5-942e-cd3e92bc5cbc.png)

**Comment by echeloni2p** [comment]
Hi

yeah, system can be slower sometimes, wait 1-3 days and photos are available.

**Comment by gyllen** [comment]
Is this still an issue?

**Comment by talllguy** [comment]
I haven't noticed it to be an error lately, no.
On Sun, Apr 10, 2016 at 21:00 Johan Gyllenspetz <notifications@github.com>
wrote:

> Is this still an issue?
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly or view it on GitHub
> <https://github.com/mapillary/mapillary_issues/issues/1719#issuecomment-208104928>
>
-- 
Elliott Plack
http://elliottplack.me

**Comment by gyllen** [comment]
Closing then, thanks for feedback

**Comment by ToeBee** [comment]
I just got this error in the middle of a sequence I uploaded recently.

Start at this picture: http://www.mapillary.com/map/im/E7pED3DDkR_KPqrddxtAew/photo

Now press the "Play" button. The "Image not found" error pops up immediately. Interestingly, if you use the "step forward" button instead of the play button this error does not appear.

**Comment by gyllen** [comment]
Is this one missing? @ToeBee great sequence by the way ;)

**Comment by ToeBee** [comment]
Oh hold on... if you lock the sequence and step to the next picture then you **do** get the error. So did that image just get swallowed by a black hole during processing or something?

**Comment by gyllen** [comment]
We have seen this happening in rare cases. Good news is that we do have the data so we are running regular repair jobs, @peterneubauer will take a look at it when he wakes up tomorrow.

**Comment by ToeBee** [comment]
Someone appears to have slept in late :)

I am currently getting this on another sequence that I just uploaded. The sequence shows in my profile and I the number of still processing images listed on my profile is down to its usual 5 (separate issue I should probably bug someone about) but I haven't gotten the email saying it is on the map yet so I'm guessing it is actually still processing despite what my profile page claims.

**Comment by ToeBee** [comment]
I got the email a little while after the last comment but now, 40 minutes since the email, the sequence is still giving the "Image doesn't exist" error:
http://www.mapillary.com/map/im/g8hZsvCerY_J7pzWYygL7w/photo

**Comment by gyllen** [comment]
@ToeBee Am I correct if the sequence does show up now?

**Comment by ToeBee** [comment]
The one from this morning works now. The one from a couple days ago is still broken.

**Comment by ToeBee** [comment]
Here is another one that got stuck tonight. I have already viewed and edited blurs on at least 3 sequences that were uploaded after this one but this one is still saying it doesn't exist: http://www.mapillary.com/map/im/2HTsdTqv6L5VYeTkhnFolA/photo

Also, the original problem I reported in this thread still isn't fixed. I was going to tweet that sequence to the national park service for National Park Week which is this week but I can't do that if it is throwing errors :( That one is http://www.mapillary.com/map/im/E7pED3DDkR_KPqrddxtAew/photo

**Comment by gyllen** [comment]
@ToeBee Can I ask what the source for you images are? Are you using manual upload? Trying to pinpoint where the problem could be.

**Comment by gyllen** [comment]
It does seems like http://www.mapillary.com/map/im/E7pED3DDkR_KPqrddxtAew/photo is working now though (we did a couple of reruns this weekend)

**Comment by ToeBee** [comment]
Oh, sorry I thought the URL in my browser had already advanced... so it isn't actually ```E7pED3DDkR_KPqrddxtAew``` but rather the next picture in that sequence that is having the problem. I can't get the picture ID for that one but it is still erroring out now.

The picture after ```E7pED3DDkR_KPqrddxtAew``` was captured and uploaded using the Mapillary app.

```g8hZsvCerY_J7pzWYygL7w``` and ```2HTsdTqv6L5VYeTkhnFolA``` are both gopro pictures that were uploaded in a browser using the manual upload page.

**Comment by ToeBee** [comment]
I just got an email stating that 1 picture in 1 sequence had just gone live for ```2HTsdTqv6L5VYeTkhnFolA```. That picture now loads but now the second picture in that sequence isn't loading either. I noticed that the "still processing" number on my profile went from 22 to 21. Before this weekend it had been sitting at 5 for several months. So I'm guessing these "pictures is missing" errors are all pictures that have gotten stuck in the processing queue for some reason. I bet if you clear all the "still processing" pictures on my account, they will all be fixed.

**Comment by gyllen** [comment]
@ToeBee yeah thats what we are working on piece by piece. There are about 6k of them in total (of about 56M ;) )

**Comment by Mercator72** [comment]
Hi, just to let you know that I have exactly the same problem. However my "still processing counter" is at 0, I received also an email, but for example the photo https://www.mapillary.com/map/im/ueb1sVZAyMKI-JXDCRtK0g/photo displays "Image not found". They were uploaded yesterday (via the iOS app), perhaps I have to wait a little bit more ?

**Comment by gyllen** [comment]
@Mercator72 This was a temporary bug in our API, it should now be fixed

**Comment by Mercator72** [comment]
Indeed I can see the images now.

**Comment by peterneubauer** [comment]
So, this is due to us sending notifications too early. I am in the process of fixing this now.

**Comment by de-vries** [comment]
@peterneubauer Probably related: 
Two days ago I uploaded some photos, for a long time nothing happened. then they showed up in my activity feed, but weren't accessible (Image not found). Now they are viewable, but they're not  on my uploads page and not visible on the map only viewable if you know were to click, like this one: http://www.mapillary.com/map/im/NZi0UPog-4OxlRxDztNEeg/photo

Photos I just uploaded are already visible on the map and on my uploads page.
